p11-kit/export-object.c

```diff
@@ -163,28 +419,37 @@ export_object (const char *token_str)
 	}
 
 	p11_kit_iter_begin (iter, modules);
-	while (p11_kit_iter_next (iter) == CKR_OK) {
-		rv = p11_kit_iter_get_attributes (iter, &attr, 1);
-		if (rv != CKR_OK) {
-			p11_message (_("failed to retrieve attribute of an object"));
+	if (p11_kit_iter_next (iter) != CKR_OK) {
+		p11_message (_("no matching object"));
```

Comment:
I would prefer this error message to be consistent across other commands. Either use the one I have been using in other commands like in delete-object or change the other ones to this one.

Reply:
I have opened an MR to adjust the other error messages